reps/2024-04-24-ray-obs-support.md

```diff
@@ -0,0 +1,103 @@
+## Summary
+
+### General Motivation
+
+Ray users can use the submit_job Api to send jobs to the ray cluster.
+
+At the same time, they can specify the location of the project's working directory in a remote file system, for example s3.
+
+This document suggests expanding the list of supported file systems and adding support for OBS [(Object Storage Service)](https://support.huaweicloud.com/intl/en-us/productdesc-obs/en-us_topic_0045853681.html).
+
+After the implementation of this improvement, the user will be able to run submit tasks specifying the location of the working directory in OBS.
+
+
+The UserCase of submiting ray job via obs is shown in Figure 1:
+
+- First, the user startups the remote ray cluster, and then upload the codes to the  obs;
+
+- Second, the user submits ray job to OBS using the submit_job APIs, as well as setting the obs path in the submit_job API
+
+- Third, after submitting the ray job, the ray cluster automatically downloads, uncompresses, and executes the codes from the OBS
+
+![Figure 1](2024-04-24-ray-obs-support/fig1.png)
```

Comment:
You need to do `<img style="background-color:white" src="2024-04-24-ray-obs-support/fig1.png">` to show the image.

Reply:
fixed